frontend/src/components/task/task-payment-form.js

```diff
@@ -16,7 +16,133 @@ import {
   Collapse
 } from '@material-ui/core'
 
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
```

Comment:
There's an error when I try to build this:
```
Module not found: Error: Can't resolve '@fortawesome/free-solid-svg-icons' in '/tmp/build_14287f47d6390cbc7db011d560d915b3/frontend/src/components/task'
        @ ./src/components/task/task-payment-form.js 23:25-69
```

Have you included in `package.json?`